src/org/usfirst/frc/team1076/robot/subsystems/FrontBackMotors.java

```diff
@@ -27,15 +27,15 @@ public FrontBackMotors(SpeedController frontMotor, SpeedController backMotor) {
 	 * @param speed a number in the range -1 to 1 inclusive.
 	 */
 	public void setFrontSpeed(double speed) {
-		this.frontMotor.set(speed);
+		this.frontMotor.set(speed * 0.5);
 	}
 
 	/**
 	 * Sets the speed of the back motor.
 	 * @param speed a number in the range -1 to 1 inclusive.
 	 */
 	public void setBackSpeed(double speed) {
-		this.backMotor.set(speed);
+		this.backMotor.set(speed * -0.5);
```

Comment:
Why is this value negative? I'm assuming because the motor is on backwards on the robot, but you may want to add a comment explaining this.